### DISCUSSION TITLE
expand and dont get all fields from it

### DISCUSSION THREAD

**Discussion by mehradamiri** [pocketbase/pocketbase#2969] [Q&A] [answered]
hi Im new in Pocketbase and I need help 
it's my code 
```js
await locals.pb.collection("users").getFullList({
      filter: `username='${username}'`,
      expand: "blogs",
    })
```
when I run it I will get all fields of blogs but I need name and description from it

**Comment by ganigeorgiev** [comment, marked as answer]
You can use the `fields` query parameter using the same format as in the json response, but keep in mind that at the moment we don't support wildcard (`*`) and you'll have to specify the exact fields you want to return. For example:

```js
await locals.pb.collection("users").getFullList({
      filter: `username='${username}'`,
      expand: "blogs",
      fields: "id,username,expand.blogs.name,expand.blogs.description"
})
```
